scripts/zones/Uleguerand_Range/mobs/Mountain_Worm_NM.lua

```diff
@@ -0,0 +1,29 @@
+-----------------------------------
+-- Area: Uleguerand Range
+--  Mob: Mountain Worm NM
+--  https://www.bg-wiki.com/ffxi/Mountain_Worm_(NM)
+-- TODO allow deaggro based on distance (core CMobEntity::CanDeaggro() forces NM and Battlefield mobs to never stop chasing)
+-- TODO confirm if despawns on lost aggro?
```

Comment:
I can help here!

He doesn't depop, but he will move very likely move (just standard random worm movement behavior) to a diff part of the cave system (little jerk...) and be really difficult to find again before he full heals.

Reply:
Ah so he can deaggro (unlike most NM) but doesn't despawn just hard to find again

Reply:
All non battlefield NM on retail can deaggro. It just they have to go a long while without interacting with you, for worms thats easy - be out of range. for other NMs they chase your \*ss down and slap you :)

Reply:
🤦‍♂️